integration-cli/docker_cli_build_test.go

```diff
@@ -1977,26 +1973,20 @@ func TestBuildCancelationKillsSleep(t *testing.T) {
 	}
 	defer ctx.Close()
 
-	var wg sync.WaitGroup
-	defer wg.Wait()
-
```

Comment:
Haven't thought about that, I thought was to better synchronize the goroutine with the external context and thought that wouldn't be needed anymore. Cool, I'll add that back then.